x-pack/plugins/index_management/__jest__/client_integration/home/home.helpers.ts

```diff
@@ -38,10 +39,15 @@ export const setup = async (): Promise<HomeTestBed> => {
     testBed.find(tab).simulate('click');
   };
 
+  const showHiddenIndices = () => {
```

Comment:
This is a toggle, so clicking on it will either show or hide hidden indices depending on its original state. I think naming this `toggleHiddenIndices` would reflect this dynamic behavior.